dlgr/griduniverse/experiment.py

```diff
@@ -1720,8 +1720,117 @@ def analyze(self, data):
         return json.dumps({
             "average_payoff": self.average_payoff(data),
             "average_score": self.average_score(data),
+            "number_of_actions": self.number_of_actions(data),
+            "average_time_to_start": self.average_time_to_start(data),
         })
 
+    def ssplit2(self, seq, splitters):
```

Comment:
I don't know about others, but I'd be tempted to use the itertools based suggestion in that answer as it's much easier to read and only very marginally slower:

    def isplit(iterable,splitters):
        return [
            list(g)
            for k,g in itertools.groupby(iterable, lambda x:x in splitters)
            if k not in splitters
        ]

Reply:
That is a fair point.

Reply:
Reverted back to ssplit2 again, see my last comment.